### ISSUE TITLE
Scale of fbp reconstruction

### ISSUE THREAD

**Issue by RingoIngo** [odlgroup/odl#1519]
I am using odl 0.7.0 and ran the filtered back projection [example](https://github.com/odlgroup/odl/blob/master/examples/tomo/filtered_backprojection_parallel_2d.py)  for the parallel 2D geometry. Somehow the output of the fbp is not scaled correctly (see screenshot). 
![Bildschirmfoto 2019-08-01 um 15 24 44](https://user-images.githubusercontent.com/10393893/62297043-b656f600-b470-11e9-9aa9-4cbb2114e59f.png)

The phantom (not visible on screenshot) takes values in `[0, 1]` while the fbp reconstruction takes values in `[0, 0.02]` (see colorbar in screenshot) the resulting error is of course large (second plot on screenshot). I think this is connected to the choice of `min_pt` and `max_pt` for `reco_space` which also influence the scale of the sinogram. Maybe I don't understand the use of `min_pt` and `max_pt` but I think that its choice should not influence the scale of the sinogram (only `shape` should)? This behaviour was the reason I tried the example in the first place.

**Comment by adler-j** [comment]
Have you tried running with the most recent ODL (here at github)? The backend (ASTRA?) has changed scalings lately which could cause this.

**Comment by RingoIngo** [comment]
Yes the backend I used was `astra_cpu` and this scaling does not happen with `skimage`. Thanks for the hint. I tried using `pip install https://github.com/odlgroup/odl/archive/master.zip` but still ran into #1498.

**Comment by adler-j** [comment]
Ok that's a bit weird. I'm sadly not the one who solved that issue the last time but perhaps @kohr-h has some input?

**Comment by kohr-h** [comment]
@RingoIngo I just tested installation via zip file, and it worked fine. Could you please give some details about your system, as well as the exact error message?
As for the original issue, which version of the ASTRA toolbox did you use?

**Comment by kohr-h** [comment]
What's the status with this issue?

**Comment by zhihan0210** [comment]
I met the same problem of scaling. I tried the code of spectral CT in odlworkshop. My odl version is '1.0.0.dev0' and my astra version is '2.1.0'
